store/src/test/java/org/apache/rocketmq/store/dledger/DLedgerMultiPathTest.java

```diff
@@ -46,8 +46,8 @@ public void multiDirsStorageTest() throws Exception {
             DefaultMessageStore dLedgerStore = createDLedgerMessageStore(base, group, "n0", peers, multiStorePath, null);
             Thread.sleep(2000);
             doPutMessages(dLedgerStore, topic, 0, 1000, 0);
-            Assert.assertEquals(11, dLedgerStore.getMaxPhyOffset()/dLedgerStore.getMessageStoreConfig().getMappedFileSizeCommitLog());
             Thread.sleep(500);
+            Assert.assertEquals(11, dLedgerStore.getMaxPhyOffset()/dLedgerStore.getMessageStoreConfig().getMappedFileSizeCommitLog());
```

Comment:
Nice change here. I guess this is something that causes our CI to fail sometimes. Hope it works. How about making this a separate PR so that we could maybe merge it faster?

Reply:
> Nice change here. I guess this is something that causes our CI to fail sometimes. Hope it works. How about making this a separate PR so that we could maybe merge it faster?

@tsunghanjacktsai  Good suggestion, I'll deal with it later.